internal/cmd/kafka/command_cluster_describe.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/confluentinc/cli/internal/pkg/resource"
 )
 
-var basicDescribeFields = []string{"IsCurrent", "Id", "Name", "Type", "NetworkIngress", "NetworkEgress", "Storage", "ServiceProvider", "Availability", "Region", "Status", "Endpoint", "RestEndpoint"}
+var basicDescribeFields = []string{"IsCurrent", "Id", "Name", "Type", "IngressLimit", "EgressLimit", "Storage", "ServiceProvider", "Availability", "Region", "Status", "Endpoint", "RestEndpoint"}
```

Comment:
Is renaming fields a breaking change?

Reply:
No, these aren't user facing. They're just used in the code.